LBH.AdultSocialCare.Api/V1/Controllers/StatusController.cs

```diff
@@ -80,6 +102,9 @@ public async Task<ActionResult<IList<Status>>> GetAll()
             }
         }
 
+        [ProducesResponseType(typeof(bool), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
         [HttpDelete]
         [Route("{statusId}")]
         public async Task<ActionResult<bool>> Delete(Guid statusId)
```

Comment:
Delete can have not found in the response. Not added to xml doc

Reply:
Thanks!